Conflicting implementations on master

Currently on master I get:
```
error[E0119]: conflicting implementations of trait `std::hash::Hash` for type `zkp::Tag`
   --> /home/casatta/.cargo/registry/src/github.com-1ecc6299db9ec823/secp256k1-zkp-sys-0.4.0/src/zkp.rs:422:1
    |
422 | impl_array_newtype!(Tag, c_uchar, 32);
    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ conflicting implementation for `zkp::Tag`
...
437 | impl hash::Hash for Tag {
    | ----------------------- first implementation here
    |
    = note: this error originates in the macro `impl_array_newtype` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0119]: conflicting implementations of trait `std::hash::Hash` for type `zkp::PedersenCommitment`
   --> /home/casatta/.cargo/registry/src/github.com-1ecc6299db9ec823/secp256k1-zkp-sys-0.4.0/src/zkp.rs:458:1
    |
458 | impl_array_newtype!(PedersenCommitment, c_uchar, 64);
    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ conflicting implementation for `zkp::PedersenCommitment`
...
473 | impl hash::Hash for PedersenCommitment {
    | -------------------------------------- first implementation here
    |
    = note: this error originates in the macro `impl_array_newtype` (in Nightly builds, run with -Z macro-backtrace for more info)

For more information about this error, try `rustc --explain E0119`.
error: could not compile `secp256k1-zkp-sys` due to 2 previous errors
warning: build failed, waiting for other jobs to finish...
error: build failed

```

a temporary workaround is to pin secp256k1-sys in Cargo.toml
```
secp256k1-sys = "=0.4.1"
```

I am not sure what the right fix is but probably it needs a release of secp256k1-sys ?

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,0 +1,7 @@
+# 0.19 - 2022-04-30 "The Taproot Release"
+
+- Taproot support for complex taptrees compatible with elements taproot signature.
+- Taproot psbt support with BIP 371
+hash. Refer to spec [here](https://github.com/ElementsProject/elements/blob/master/doc/taproot-sighash.mediawiki)
+- Support for new tapscript transaction introspection opcodes as per the [spec](https://github.com/ElementsProject/elements/blob/master/doc/tapscript_opcodes.md).
+- Works with bitcoin 0.28 key types.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "elements"
-version = "0.18.0"
+version = "0.19.0"
 authors = ["Andrew Poelstra <apoelstra@blockstream.com>"]
 description = "Library with support for de/serialization, parsing and executing on data structures and network messages related to Elements"
 license = "CC0-1.0"

